appinfo/app.php

```diff
@@ -32,11 +32,3 @@
 if ($gssMode === 'master') {
 	return;
 }
-
-$userSession = \OC::$server->getUserSession();
-$masterUrl = $config->getSystemValue('gss.master.url', '');
-if (!$userSession->isLoggedIn() && !empty($masterUrl) &&
-	\OC::$server->getRequest()->getPathInfo() === '/login') {
-	header('Location: '. $masterUrl);
-	exit();
-}
```

Comment:
Ummm ... removing this could lead to weird stuff for users that get a share on another node of the instance. Thus as redirect to the master instance is crucial. We should add a flag to stuff the redirect

Reply:
Appending something like `/login?admin` will stop the redirect. In this way the normal user experience is far better, but still allows admins to login. Does this sound like a plan?

Reply:
don't think so. This wasn't there until my last pull request. I added it as I thought it might be nice but it turned out to cause some other problems potentially. Everybody who already uses the global site selector uses it without this code.

Reply:
Okay - I just also had this in the previous version of the GS mechanism. Thus I was curious, but couldn't find the code there neither. Let's keep it out then and see how it works 👍